src/Common/parseRemoteDescription.cpp

```diff
@@ -82,8 +82,8 @@ std::vector<String> parseRemoteDescription(const String & description, size_t l,
         /// Either the numeric interval (8..10) or equivalent expression in brackets
         if (description[i] == '{')
         {
-            int cnt = 1;
-            int last_dot = -1; /// The rightmost pair of points, remember the index of the right of the two
+            ssize_t cnt = 1;
```

Comment:
Yes it can. I prefer to use ssize_t/size_t for standard (read: non db specific) code like this one, but if you thing Int64 should be used here, fine by me, let me know.

Reply:
`ssize_t` is okay for now. I was maybe sceptical because I see this type used quite rarely in ClickHouse.